Business apps tutorial (#628)
SimpleDataLabsInc/prophecy-docs: docs/analysts/business-apps/business-apps.md

tags: []
---

> Business applications make it easy for non-technical users to run data pipelines through intuitive, form-based interfaces—no engineering required. Instead of writing or modifying pipeline code, users can rely on pre-built, templated apps to run pipelines quickly and consistently.

Reviewer: ```suggestion
Business applications simplify the process for non-technical users to run data pipelines through intuitive, form-based interfaces—no engineering required. Instead of writing or modifying pipeline code, users can rely on pre-built, templated apps to run pipelines quickly and consistently.
```
Nit: I typically avoid words like easy because I think it's subjective to the person, and it will make a user feel terrible if they can't figure it out with our docs.

Author: i agree. thanks for the reminder!

These reusable apps help teams collaborate more effectively by eliminating duplicate work and standardizing processes across users. Built-in logic and guardrails further reduce the risk of errors, guiding users through safe, tested execution paths.

## App capabilities

> Business apps look and feel like simple forms, but they pack powerful capabilities. Learn more in this [guide](/analysts/run-apps).

Reviewer: ```suggestion
Business apps look and feel like simple forms, but they pack the following powerful capabilities:
```
Suggestion: I'd move the "Learn more" part to the bottom of the section.

Author: smart

- **Assign values to pipeline parameters**  
  Set variables such as dates, thresholds, or regions at runtime. Parameters appear as form fields, making it easy to customize each run without writing code.

- **View and download app-generated output data**  
  Explore and download the outputs produced by the pipeline after the app runs. This lets you verify results before sharing, without digging into external storage systems.

- **Save and reuse app configurations**  
  Save a set of parameter values and inputs as a configuration. Reuse it anytime to run the same job again for recurring tasks or standardized reports.

- **Schedule automated app runs**  
  Set up your app to run on a regular basis to automate recurring workflows.

## Working with business apps

Business apps live in the **App Directory**, where each app is linked to a specific data pipeline. You’ll see apps from any project that has been shared with your team, making it easy to discover and reuse trusted workflows. Apps always run against the latest published version of their pipeline, so you can count on up-to-date logic and data sources.

> Want to build your own app? Turn your pipelines into simple, form-based tools that anyone on your team can use. Follow our guide to [create a business app](/analysts/create-business-applications).

Reviewer: ```suggestion
If you want to build an app and turn your pipelines into simple, form-based tools that anyone on your team can use, follow our guide to [create a business app](/analysts/create-business-applications).
```
